zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RequestMapper.java

```diff
@@ -243,6 +245,16 @@ public static Either<ProblemDetail, UpdateJobRequest> toJobUpdateRequest(
                     updateRequest.getChangeset().getTimeout())));
   }
 
+  public static Either<ProblemDetail, UpdateRoleRequest> toRoleUpdateRequest(
```

Comment:
🔧 The `Either` is not needed if there is no case that leads to a `Left`. We can just return the `UpdateRoleRequest` if we don't do any validation.

Reply:
I chose Either for consistency with the other mapping functions, but I now see that there's already exceptions. I'll change it.

Reply:
I actually ended up adding validation, so it's back to Either now.